apps/mobile/src/features/browser/approver-sheet/stx/get-addresses/get-addresses.tsx

```diff
@@ -0,0 +1,86 @@
+import { useRef, useState } from 'react';
+
+import { AccountSelectorSheet } from '@/features/accounts/account-selector/account-selector-sheet';
+import { GetAddressesApproverLayout } from '@/features/approver/layouts/get-addresses.layout';
+import { useAccounts } from '@/store/accounts/accounts.read';
+import { userConnectsApp } from '@/store/apps/apps.write';
+import { App } from '@/store/apps/utils';
+import { useStacksSigners } from '@/store/keychains/stacks/stacks-keychains.read';
+import { useAppDispatch } from '@/store/utils';
+import { bytesToHex } from '@stacks/common';
+
+import {
+  RpcRequest,
+  RpcResponse,
+  createRpcSuccessResponse,
+  stxGetAddresses,
+} from '@leather.io/rpc';
+import { SheetRef } from '@leather.io/ui/native';
+
+interface GetAddressesApproverProps {
+  app: App;
+  message: RpcRequest<typeof stxGetAddresses>;
```

Comment:
```suggestion
  request: RpcRequest<typeof stxGetAddresses>;
```

We used `message` in extension a lot in past, but this was really a relic from pre-following RPC correctly. This was updated recently.

Suggest only using `request` to keep consistent with RPC terminology.